src/Windows/Markdown/Editor/Impl/Preview/Code/EvalSession.cs

```diff
@@ -43,7 +43,7 @@ internal sealed class EvalSession : IDisposable {
 
             if (!RSession.IsHostRunning) {
                 var settings = _services.GetService<IRSettings>();
-                await RSession.EnsureHostStartedAsync(new RHostStartupInfo(), SessionCallback, 3000, ct);
+                await RSession.EnsureHostStartedAsync(new RHostStartupInfo(isInteractive: true), SessionCallback, 3000, ct);
```

Comment:
It has to be interactive so it behaves like REPL and outputs errors without terminating the host.